include/bluetooth/services/ddfs.h

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef BT_DDFS_H_
+#define BT_DDFS_H_
+
+#include <stdlib.h>
+#include <zephyr/types.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/uuid.h>
+#include <bluetooth/gatt.h>
+
+/**@file
+ * @defgroup bt_ddfs Direction and Distance Finding Service API
+ * @{
+ * @brief API for the Direction and Distance Finding Service (DDFS).
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @brief UUID of the DDF Service. **/
+#define BT_UUID_DDFS_VAL \
+	BT_UUID_128_ENCODE(0x21490000, 0x494a, 0x4573, 0x98af, 0xf126af76f490)
+
+/** @brief UUID of the Distance Measurement Characteristic. **/
+#define BT_UUID_DDFS_DISTANCE_MEAS_VAL \
+	BT_UUID_128_ENCODE(0x21490001, 0x494a, 0x4573, 0x98af, 0xf126af76f490)
+
+/** @brief UUID of the Azimuth Measurement Characteristic. **/
+#define BT_UUID_DDFS_AZIMUTH_MEAS_VAL \
+	BT_UUID_128_ENCODE(0x21490002, 0x494a, 0x4573, 0x98af, 0xf126af76f490)
+
+/** @brief UUID of the Elevation Measurement Characteristic. **/
+#define BT_UUID_DDFS_ELEVATION_MEAS_VAL \
+	BT_UUID_128_ENCODE(0x21490003, 0x494a, 0x4573, 0x98af, 0xf126af76f490)
+
+/** @brief UUID of the Feature Characteristic. **/
+#define BT_UUID_DDFS_FEATURE_VAL \
+	BT_UUID_128_ENCODE(0x21490004, 0x494a, 0x4573, 0x98af, 0xf126af76f490)
+
+/** @brief UUID of the Control Point Characteristic. **/
+#define BT_UUID_DDFS_CTRL_POINT_VAL \
+	BT_UUID_128_ENCODE(0x21490005, 0x494a, 0x4573, 0x98af, 0xf126af76f490)
+
+#define BT_UUID_DDFS                 BT_UUID_DECLARE_128(BT_UUID_DDFS_VAL)
+#define BT_UUID_DDFS_DISTANCE_MEAS   BT_UUID_DECLARE_128(BT_UUID_DDFS_DISTANCE_MEAS_VAL)
+#define BT_UUID_DFFS_AZIMUTH_MEAS    BT_UUID_DECLARE_128(BT_UUID_DDFS_AZIMUTH_MEAS_VAL)
+#define BT_UUID_DFFS_ELEVATION_MEAS  BT_UUID_DECLARE_128(BT_UUID_DDFS_ELEVATION_MEAS_VAL)
+#define BT_UUID_DDFS_FEATURE         BT_UUID_DECLARE_128(BT_UUID_DDFS_FEATURE_VAL)
+#define BT_UUID_DFFS_CTRL_POINT      BT_UUID_DECLARE_128(BT_UUID_DDFS_CTRL_POINT_VAL)
+
+/** Ranging mode */
+enum bt_ddfs_dm_ranging_mode {
+	/* Round trip timing */
```

Comment:
I think doc string should start with /** here to be generated by doxygen and to be consistent with other enums in BLE headers

Reply:
Fixed.